tutorial/connect-from-aks/ConnectFromAKS/Program.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Azure.Identity;
+using StackExchange.Redis;
+using static System.Console;
+
+WriteLine("This sample shows how to connect to an Azure Redis cache using managed identity or access keys.");
+try
+{
+    var connectionOption = Environment.GetEnvironmentVariable("CONNECTION_OPTION");
+    var cacheHostName = Environment.GetEnvironmentVariable("CACHE_HOSTNAME");
+    ConfigurationOptions? configurationOptions = null;
+
+    switch (connectionOption)
+    {
+        case "MANAGED_IDENTITY":
+            WriteLine($"Connecting to {cacheHostName} with managed identity..");
+            configurationOptions = await ConfigurationOptions.Parse($"{cacheHostName}:6380").ConfigureForAzureWithTokenCredentialAsync(new DefaultAzureCredential());
+            configurationOptions.AbortOnConnectFail = false; // Fail fast for the purposes of this sample. In production code, this should remain false to retry connections on startup
+            break;
+
+        case "ACCESS_KEY":
+            WriteLine("Connecting to {cacheHostName} with an access key..");
+            var connectionString = Environment.GetEnvironmentVariable("CONNECTION_STRING");
```

Comment:
For symmetry with the workload identity option, can we just require a CACHE_HOSTNAME plus a REDIS_ACCESSKEY, rather than a CONNECTION_STRING?

Reply:
Changed to using access key and also configurable port number